src/FunderMaps.Core/Extensions/ExceptionExtensions.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AutoMapper
```

Comment:
If you extend a class/method then the extensions must be contained within the same namespace. Otherwise extensions require additional (hidden) namespaces. This code does not extend `AutoMapper`. See the C# docs